src/components/Inputs/Input.types.ts

```diff
@@ -209,6 +209,17 @@ export interface InputProps<T>
      * @default false
      */
     disabled?: boolean;
+    /**
+     * Make input expandable on focus.
+     * Default slides from left upon setting this to true.
+     * @default false
+     */
+    expandable?: boolean;
+    /**
+     * Make input expandable slide from right.
+     * @default false
+     */
+    expandRight?: boolean;
```

Comment:
@dkilgore-eightfold thoughts on this naming from a RTL perspective?

Reply:
This is fine, but this feature still does not yet support canvas direction. Added comments in CSS. This prop will not work for loc.